openmetadata-ui/src/main/resources/ui/cypress/common/Entities/EntityClass.ts

```diff
@@ -101,10 +101,11 @@ export const glossaryDetails2 = {
   tags: [],
   mutuallyExclusive: false,
 };
+const glossaryTermDetails1Name = `CypressBankNumber-${uuid()}`;
 
 export const glossaryTermDetails1 = {
-  name: `CypressBankNumber-${uuid()}`,
-  displayName: 'Cypress BankNumber',
+  name: glossaryTermDetails1Name,
+  displayName: glossaryTermDetails1Name,
```

Comment:
I have kept it different intentionally, wanted to search using displayName and it should gave correct results.
Can we try to test that functionality here?

Reply:
Thanks for the insights, I will surely have look!